app/components/post_list/post/body/content/youtube/youtube.tsx

```diff
@@ -89,7 +89,7 @@ const YouTube = ({googleDeveloperKey, intl, isReplyPost, post}: YouTubeProps) =>
 
         if (Platform.OS === 'ios') {
             YouTubeStandaloneIOS.
-                playVideo(videoId, startTime).
+                playVideo(videoId).
```

Comment:
this should not change.. here you are removing a feature for the YouTube videos.. please exclude